### DISCUSSION TITLE
SCL Accessor

### DISCUSSION THREAD

**Discussion by bcerripromethee** [sertit/eoreader#149] [Q&A] [answered]
Hey,
Is there an accessor to the SCL image for Sentinel-2 sensor (Scene Classification Map) ? This one can be found in the `...\GRANULE\L2A_T31TFJ_A036856_20220713T103035\IMG_DATA\R20m` folder

**Comment by remi-braun** [comment]
Hi!
Right now, I never faced the need to access it, so sadly no 😅

**Reply by bcerripromethee** [reply]
Argh, would it be possible to add it ?

**Reply by remi-braun** [reply, marked as answer]
I created an issue (#153) for this request :)
